database/src/pool/postgres.rs

```diff
@@ -1185,13 +1199,13 @@ where
             Some(aid) => aid.get::<_, i32>(0) as u32,
             None => {
                 self.conn()
-            .query_opt("insert into artifact (name, date, type) VALUES ($1, $2, $3) ON CONFLICT DO NOTHING RETURNING id", &[
-                &info.name,
-                &info.date,
-                &info.kind,
-            ])
-            .await
-            .unwrap();
+                    .query_opt("insert into artifact (name, date, type) VALUES ($1, $2, $3) ON CONFLICT DO NOTHING RETURNING id", &[
```

Comment:
Nit; capitalisation of SQL keywords. It's vastly more readable and the style we've been using with all of the new queries.
```suggestion
                    .query_opt("INSERT INTO artifact (name, date, type) VALUES ($1, $2, $3) ON CONFLICT DO NOTHING RETURNING id", &[
```

Reply:
Huh, this was `rustfmt` reformatting some old code, interesting.